doc/reference/reference_sql/sql_statements_and_clauses.rst

```diff
@@ -651,6 +749,10 @@ Note that a referenced column should meet one of the following requirements:
 -   A referenced column has a UNIQUE constraint.
 -   A referenced column has a UNIQUE index.
 
+Note that before the :doc:`2.11.0 </release/2.11.0>` version, uniqueness of a foreign key is checked when creating a constraint (for example, using ``CREATE TABLE`` or ``ALTER TABLE``).
```

Comment:
I think it is more `existence of an index on the referenced columns` then `uniqueness of a foreign key`.

Reply:
Updated to:
```
Note that before the :doc:`2.11.0 </release/2.11.0>` version, an index existence for the referenced columns is checked when creating a constraint (for example, using ``CREATE TABLE`` or ``ALTER TABLE``).
Starting with 2.11.0, this check is weakened and existence of an index is checked during data insertion.
```